tests/sentry/web/frontend/test_auth_organization_login.py

```diff
@@ -636,9 +636,9 @@ def test_flow_authenticated_without_verified_without_password(self):
         path = reverse("sentry-auth-sso")
 
         resp = self.client.post(path, {"email": "bar@example.com"})
-        self.assertTemplateUsed(resp, "sentry/auth-confirm-identity.html")
+        self.assertTemplateUsed(resp, "sentry/auth-confirm-account.html")
         assert resp.status_code == 200
-        assert resp.context["existing_user"] is None
+        assert resp.context["existing_user"] == user
```

Comment:
I think this might now be fully redundant to [`test_flow_verify_and_link_without_password_sends_email`](https://github.com/getsentry/sentry/pull/31774/files#diff-8b70a9146a20fc2ca61609643e1c1aee1eac82c6c31a6ee75616997fbf5e715bR1052) but 🤷 might as well keep it.